src/components/ui/notifications/Notifications.vue

```diff
@@ -23,11 +23,14 @@
                 </div>
```

Comment:
[Docs](https://github.com/epicmaxco/vuestic-admin/wiki/Tooltips-and-Popovers) are dated.
Please also copy docs to component folder as shown [here](https://github.com/epicmaxco/vuestic-admin/blob/develop/.github/CONTRIBUTING.md#component-folder-structure)